Update example to a full functional application

stephanebachelier opened the pull request:
Close #19

tofumatt commented on examples/main.js:
```diff
@@ -0,0 +1,241 @@
+(function(Backbone, $, _) {
+    'use strict';
+
+    // Set driver (optional, but we use localforage here so developers
```
Should be "use localStorage" here, I think. Though Firefox devtools has an indexeddb storage inspector that's coming along awesome ;-)

tofumatt replied:
Should be "use localStorage" here, I think. Though Firefox devtools has an indexeddb storage inspector that's coming along awesome ;-)

stephanebachelier commented:
@tofumatt if you want to play with the example it's here:

http://stephanebachelier.github.io/localForage-backbone/

tofumatt commented:
Nice! I'll do a bit more review later this evening. That's awesome though
:-)

-tofumatt
On 27 Feb 2015 10:49, "Stéphane Bachelier" <notifications@github.com> wrote:

> @tofumatt <https://github.com/tofumatt> if you want to play with the
> example it's here:
>
> http://stephanebachelier.github.io/localForage-backbone/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/localForage-backbone/pull/25#issuecomment-76375548>
> .
>

stephanebachelier commented:
this is localforage which is awesome

tofumatt commented:
`examples/img/device.png` is cute but obviously an iPhone, any chance for a more generic phone outline graphic?

tofumatt commented:
Looks like the icon set you're using isn't loading? I see black boxes in Safari and Unicode weird characters in Firefox (both Mac 10.10.2)

Safari:
![screenshot 2015-03-10 17 24 19](https://cloud.githubusercontent.com/assets/90871/6580754/7da4ce4e-c74a-11e4-9d98-1b1381713a31.png)

Firefox (Developer Edition):
![screenshot 2015-03-10 17 24 34](https://cloud.githubusercontent.com/assets/90871/6580755/803f16e6-c74a-11e4-8e8f-fd024017c28f.png)

I love the different driver selection in the tabs though, that's very cool.

tofumatt commented:
Overall great! Really cool, and I think I'll use this as a general localForage example too!

Can we fix the unicode bugs and possibly use a more generic phone graphic? If so I'd say it's good to merge!

stephanebachelier commented:
@tofumatt I think the unicode bugs is due to CORS. I'll come back to you soon. For the phone I've used an iPhone because I add this image, but a Firefox OS phone image would be better :)

stephanebachelier commented:
@tofumatt I've fixed the problem about the icon and change the phone image.

I've updated both this PR and the gh-pages which means that you can verify the changes on http://stephanebachelier.github.io/localForage-backbone/.
Tested with Safari, Chrome and FF dev edition.

I'll publish a mobile application based on this example next week on the stores.

tofumatt commented:
Very cool! :+1:
